Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.11.5-alpine3.8 as build
+
+RUN apk --no-cache add git linux-headers ca-certificates
+COPY . /go/src/github.com/ChainSafeSystems/go-pre
+WORKDIR /go/src/github.com/ChainSafeSystems/go-pre
+ENV GO111MODULE=on
```

Comment:
No modules for the time being, lets remove this

Reply:
@ansermino done!